sftp/sftp.go

```diff
@@ -245,6 +259,10 @@ func NewFs(name, root string) (fs.Fs, error) {
 		Timeout:         fs.Config.ConnectTimeout,
 	}
 
+	if insecureCipher {
+		config.Config.Ciphers = []string{"aes128-cbc"}
```

Comment:
Does that mean it just uses that cipher?  Can we make it offer the other ciphers too?

Reply:
Yes I believe it does. I was in two minds about making it offer the standard ciphers. aes128-cbc seems to be the only one that's "available but disabled" in the `crypto/ssh` library. It also means connections will fail if the user attempts to connect to a server that refuses to support this insecure cipher.

Happy to have it offer this cipher in addition to the standard ones, if that's what you'd prefer.

Reply:
I think offering the standard ciphers as well as this one would be a good idea just in case people are connecting to different servers using the same remote.  Provided that works for your use case of course!

In general ssh should negotiate the most secure cipher available so don't think there is a downside to having them all in the list.

Reply:
Agreed, PR updated.